### ISSUE TITLE
Instructions (detailed imgur) are no longer available

### ISSUE THREAD

**Issue by pyrocide** [Cristov9000/SuperGameGirl#8]
I finally have all the parts required but the original imgur post is gone, archive does not include all photos making replication much more difficult.

**Comment by BioHaZard1** [comment]
This will help you out https://imgur.com/a/KuBRE#eaF57E6

I also have a complete copy of the original guide too. Not sure why the project is being rebranded and the guide got taken down. Strange.

**Comment by pyrocide** [comment]
Ah much appreciated. I started the desoldering hell and holy shit it sucks.

**Comment by BioHaZard1** [comment]
Yeah, I didn't find it too hard, look me about 30-40 minutes to remove all components. I used a solder sucker, it made the process much easier.

**Comment by mattc74** [comment]
You can view the original guide via. the wayback machine, here:

https://web.archive.org/web/20160105060629/https://imgur.com/a/gFhk7

**Comment by jrdnlc** [comment]
@BioHaZard1 You mind sharing your copy of the conplete guide?

**Comment by BioHaZard1** [comment]
@pyrocide 
@jrdnlc 

Re-Uploaded guide:
https://imgur.com/a/JKNMZ

**Comment by jrdnlc** [comment]
Thank you @BioHazard1
